### ISSUE TITLE
Plugin does not show content in the stdlib

### ISSUE THREAD

**Issue by alex-s168** [FalsePattern/ZigBrains#13]
OS: Ubuntu
yes, i have ZLS set up
yes, i have everything configured correctly

everything works except that it doesnt "detect" the stdlib

**Comment by ewrogers** [comment]
Similar for me:

- Mac OS Sonoma 14.0 (arm64)
- Zig 0.11.0
- ZLS 0.11.0
- CLion 2023.2.2 (aarch64)
- ZigBrains 0.8.1

I have no other plugins installed. I did notice GitHub CoPilot would break auto-complete at times, so I stopped using it on CLion and only have ZigBrains installed.

Restarted the IDE several times, but still nothing. My LSP can show functions from other places in my code just fine but anything in `std` doesn't work for autocomplete.

**Comment by ewrogers** [comment]
I've been looking into this more since I really can't bring myself to use VS Code just for Zig while using the JetBrains suite of IDEs for everything else.

My stubbornness seems to have paid off! I have it working with CLion 2023.2.2 + ZigBrains 0.8.1 (latest).

The issue seems to be that the `zls.json` is not added/configured by default. I'm not sure how the [official ziglang VS Code extension](https://github.com/ziglang/vscode-zig) gets around this with autocomplete working out of the box, possibly work looking into.

If you do `zls --show-config-path` you'll get something like `~/Library/Application Support/zls.json` and notice that it's missing. I looked into the [ZLS config options](https://github.com/zigtools/zls#configuration-options) and decided to try this out for mine:

```json
{
  "zig_lib_path": "/Users/erik/.zigtools/lib/zig",
  "zig_exe_path": "/Users/erik/.zigtools/zig"
}
```

Restarting CLion and now I get auto-completed for `std` stuff!

**Comment by SuperCraftAlex** [comment]
Thanks a lot!

**Comment by FalsePattern** [comment]
Fixed in 12.0.0, ZigBrains now autogenerates a config file with the zig executable and std lib you specify in the zig toolchain, and gives it to ZLS if you don't have a config file specified manually.
